docs/en/stack/ml/anomaly-detection/ml-jobs-from-visuals.asciidoc

```diff
@@ -15,9 +15,14 @@ You can create {anomaly-jobs} from the compatible Lens charts on **Dashboard**.
 `area`, `area_percentage_stacked`, `area_stacked`, `bar`, `bar_horizontal`, 
 `bar_horizontal_stacked`, `bar_percentage_stacked`, `bar_stacked`, and `line`.
 
+* Supported {anomaly-detect} functions are `average`, `count`, `max`, `median`, 
+`min`, `sum`, `unique_count`.
+
 * The chart must contain a date field on one axis and it must be the same as the 
 default date field for the {data-source}.
 
+* Charts containing time shifts are not supported.
```

Comment:
also per field "filter by" adjustments are now allowed.
I'm not sure of the best way to describe these.

<img width="406" alt="image" src="https://user-images.githubusercontent.com/22172091/184178599-b41d98c2-2e8d-4b4a-b8d6-c1238bf9f621.png">